44.md

```diff
@@ -0,0 +1,170 @@
+NIP-44
+======
+
+Encrypted Direct Message (Versioned)
+------------------------------------
+
+`optional` `author:paulmillr` `author:staab`
+
+The NIP introduces versioned encryption, allowing multiple algorithm choices to exist simultaneously.
```

Comment:
The NIP introduces a versioned encryption data model, allowing multiple algorithm choices to exist simultaneously.